Goobi/config/goobi_config.properties

```diff
@@ -115,20 +115,20 @@ useOrigFolder=true
 # to have sufficient permissions to do whatever they should do.
 #
 # For examples of these scripts, see:
-# https://github.com/kitodo/kitodo-production/tree/1.11.x/Goobi/scripts
+# https://github.com/kitodo/kitodo-production/tree/2.0.0/Goobi/scripts
```

Comment:
Linking to master / development branch is a bad idea because scripts could be change in development, changed to be incompatible, moved or deleted. I will add a notice to look for current stable tags.